### DISCUSSION TITLE
Sample UBL invoice

### DISCUSSION THREAD

**Discussion by Patilsv287** [phax/en16931-cii2ubl#21] [Q&A]
Hi There, @phax 

Where can I see how the  the invoice will be displayed after converting from CII to UBL? I want to see the structure as well as the look and feel? Are there any sample documents? Thank you in advance :)

**Comment by phax** [comment]
@Patilsv287 you can have a look at https://github.com/phax/en16931-cii2ubl/tree/master/en16931-cii2ubl/generated for the different generated UBL files.
Additionally, there is an open API at https://peppol.helger.com/public/locale-en_US/menuitem-tools-rest-api#cii2ubl where you can test the conversion live (no guarantee or SLA there...).
